releases.yml

```
@@ -12,6 +12,10 @@
       url: /docs/patterns/navigation#full-width-dropdowns
       status: New
       notes: We've introduced a new full-width dropdown option to the top navigation component.
+    - component: Tiered list pattern
```

```suggestion
    - component: Tiered list
```
[Hero PR](https://github.com/canonical/vanilla-framework/pull/5248) does not use "pattern" here, maybe we should come to group decision on how to name these in release notes? I don't think they need "pattern"

See [Hero 4.15 release notes](https://vanilla-framework-5248.demos.haus/docs/whats-new)